### PR TITLE
[VENTUS][NFC] Add inline assembly codes testcase

### PR DESCRIPTION
**Pull request by ziliangzl**
Add an inline-asm testcase for constraint r
Add an inline-asm testcase for branch instruction

### CODE REVIEW

**Review by Jules-Kong — COMMENTED** [review]
Will the current use case be run when building the toolchain?

File: llvm/test/CodeGen/RISCV/VentusGPGPU/inline-asm-branch.ll [hunk 1]
```diff
@@ -0,0 +1,39 @@
+
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: llc -mtriple=riscv32 -mcpu=ventus-gpgpu -verify-machineinstrs < %s \
+; RUN:   | FileCheck -check-prefix=VENTUS %s
+
+define ventus_kernel void @getMax(ptr addrspace(1) %out, ptr addrspace(4) %in) {
```
Comment (Jules-Kong) [inline llvm/test/CodeGen/RISCV/VentusGPGPU/inline-asm-branch.ll:6]: Why add these two testcases? Were there any changes made before? What is coomit id?

**Comment by ziliangzl** [comment]
> Will the current use case be run when building the toolchain?

No.

File: llvm/test/CodeGen/RISCV/VentusGPGPU/inline-asm-branch.ll (hunk 1 above)
Comment (ziliangzl) [reply llvm/test/CodeGen/RISCV/VentusGPGPU/inline-asm-branch.ll:6]: No changes,just verify if inline-asm is avaliable in ventus.

**Review by zhoujingya — APPROVED** [review]

**Pull request merged**